Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/index.html

```diff
@@ -49,7 +49,7 @@
     <div id="parent" class="fxs-mainwindow">
         <div id="leftDiv">
             <div></div>
-                <table id="myTable" class="table table-bordered table-condensed table-striped sortable ui-widget-content" tabindex="1">
+                <table id="myTable" class="table table-bordered table-condensed table-striped sortable ui-widget-content" tabindex="0">
```

Comment:
Please remove the `tabindex="0"` here, it will conflict with existing grid cell focus design